src/main/java/org/example/services/DeliveryEmployeeService.java

```diff
@@ -0,0 +1,33 @@
+package org.example.services;
+
+import org.example.Validators.EmployeeValidator;
+import org.example.daos.DeliveryEmployeeDao;
+import org.example.exceptions.InvalidException;
+import org.example.models.DeliveryEmployeeRequest;
+
+import java.sql.SQLException;
+
+public class DeliveryEmployeeService {
+
+    private EmployeeValidator employeeValidator;
+    private DeliveryEmployeeDao dao;
+
+    public DeliveryEmployeeService(final DeliveryEmployeeDao dao1,
+    final EmployeeValidator empValidator) {
+        this.dao = dao1;
+        this.employeeValidator = empValidator;
+    }
+
+    public int createDeliveryEmployee(
+        final DeliveryEmployeeRequest deliveryEmployee)
+            throws SQLException, InvalidException {
+        employeeValidator.validateEmployee(deliveryEmployee);
+        int id = dao.createDeliveryEmployee(deliveryEmployee);
+
+        if (id == -1) {
+            return -1;
+        }
+        return id;
```

Comment:
you can do this on one line
```suggestion
        return dao.createDeliveryEmployee(deliveryEmployee);
```